### PR TITLE
Eslint enforce consistent indentation

### PR DESCRIPTION
**Pull request by GabrielVidal1**
I added the `indent` rule to the eslint file to enforce consistent indentation.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -82,7 +83,8 @@ module.exports = {
         '@typescript-eslint/no-floating-promises': 'off',
         '@typescript-eslint/unbound-method': 'off',
         '@typescript-eslint/explicit-function-return-type': 'off',
-        '@typescript-eslint/naming-convention': 'off'
+        '@typescript-eslint/naming-convention': 'off',
+        'indent': 'off',
```
Comment (smadbe) [inline .eslintrc.js:87]: Why no indent rules for tests?

File: src/app/modules/group/components/group-manager-list/group-manager-list.component.ts [hunk 2]
```diff
@@ -42,8 +42,8 @@ export class GroupManagerListComponent implements OnChanges {
         this.managers = managers.map(manager => ({ ...manager, can_manage_as_text: this.getManagerLevel(manager) }));
         this.state = 'ready';
       },
-        _err => {
-          this.state = 'error';
-        });
+      _err => {
```
Comment (smadbe) [inline src/app/modules/group/components/group-manager-list/group-manager-list.component.ts:45]: Here, it is not really correct, but mainly as if `_err =>..` is on a new line, `(managers: Manager[]) => ` should be on a new line as well

**Review by smadbe — CHANGES_REQUESTED** [review]

File: src/app/modules/group/components/group-manager-list/group-manager-list.component.ts (hunk 2 above)
Comment (GabrielVidal1) [reply src/app/modules/group/components/group-manager-list/group-manager-list.component.ts]: fixed

File: .eslintrc.js (hunk 1 above)
Comment (GabrielVidal1) [reply .eslintrc.js]: fixed

**Review by smadbe — APPROVED** [review]

**Pull request merged**